contrib/gitian-descriptors/gitian-linux.yml

```diff
@@ -147,13 +147,6 @@ script: |
   SOURCEDIST=$(echo bitcoin-*.tar.gz)
   DISTNAME=${SOURCEDIST/%.tar.gz}
 
-  # Correct tar file order
-  mkdir -p temp
-  pushd temp
-  tar -xf ../$SOURCEDIST
-  find bitcoin-* | sort | tar --mtime="$REFERENCE_DATETIME" --no-recursion --mode='u+rw,go+r-w,a+X' --owner=0 --group=0 -c -T - | gzip -9n > ../$SOURCEDIST
```

Comment:
So this is not needed for the `git archive` output?
That's nice!

Reply:
>    * The timestamp seems to be the same for all the files, but I don't know where it comes from. Oh it's the "CommitDate" of the last commit!

Yes. From [`git archive` docs](https://git-scm.com/docs/git-archive):
> `git archive` behaves differently when given a tree ID versus when given a **commit ID** or tag ID. In the first case the current time is used as the modification time of each file in the archive. In the latter case **the commit time as recorded in the referenced commit object is used** instead.

Reply:
Seems this is the same for the `mtime` in the global header.